recipes/pyfoomb/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "pyfoomb" %}
+{% set version = "2.17.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MicroPhen/pyFOOMB/archive/v{{ version }}.tar.gz
+  sha256: 23d970bf7e41cd1f7b90aede6d306ff0781f1d404fe257f1185537e1d1a52214
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```

The Python version constraint should match the one in run
```suggestion
    - python >=3.7
```